vizro-core/src/vizro/static/js/models/dashboard.js

```diff
@@ -1,9 +1,27 @@
+var maxWidth = screen.availWidth;
+var maxHeight = screen.availHeight;
+
 export function _update_dashboard_theme(checked) {
   return checked ? "vizro_light" : "vizro_dark";
 }
 
 export function _collapse_nav_panel(n_clicks, is_open) {
-  if (!n_clicks) return dash_clientside.no_update;
+  if (!n_clicks) {
+    /* Automatically collapses left-side if xs and s-devices are detected*/
+    if (maxWidth < 576 || maxHeight < 576) {
```

Comment:
```suggestion
    if (window.innerWidth < 576 || window.innerHeight < 576)  {
```
This will make it so that if you start in a smaller screen on a desktop, the left side will be collapsed.

Reply:
Ah yes - thank you!!! 🚀